scripts/test262-runner.js

```diff
@@ -1263,7 +1264,13 @@ function getBanners(test: TestFileInfo, fileContents: string): ?BannerData {
     } else if (bannerText.includes("attribute of 'arguments'")) {
       return null;
     } else if (bannerText.includes("poisoned")) return null;
-    data = yaml.safeLoad(banners[0].slice(3, -3));
+    try {
+      data = yaml.safeLoad(banners[0].slice(3, -3));
+    } catch (e) {
+      // Some versions of test262 have comments inside of yaml banners.
+      // parsing these will usually fail.
```

Comment:
Is there a list of test files that trigger this? We need to fix those in test262!

Reply:
I think they may be fixed already. We need to update our reference to the tests. This is a bit involved because Facebook internal repositories need to get updated as well. Let me see if I can get someone to take an interest.